Lombiq.NodeJs.Extensions/Lombiq.NodeJs.Extensions.targets

```
@@ -15,6 +15,8 @@
       SCSS_SOURCE=$(NodeJsExtensionsStylesSourceFolder.Replace('\', '/'));
       SCSS_TARGET=$(NodeJsExtensionsStylesTargetFolder.Replace('\', '/'));
     </NpmCommandsEnvironmentVariables>
+    <!-- The importance with which to print informational messages. -->
+    <Importance>High</Importance>
```

Shouldn't this be
```suggestion
    <Importance Condition="'$(Importance)' = ''">High</Importance>
```
so it's only set to "High" if the user didn't already provide a value in the csproj?

---

I'll add an underscore to convey that this is an internal flag. And I'll set it to `Normal`, too. Useful for debugging, but not much else.